src/vkbd/usb-gadget.h

```diff
@@ -0,0 +1,152 @@
+#ifndef _USBGADGET_H_
+#define _USBGADGET_H_
+#define _USBGADGET_H_
+
+#define HID_ALT_GR 0x40
+#define HID_CTRL 0x1
+#define HID_SHIFT 0x2
+#define HID_SUPER 0x8
+#define HID_ALT 0x4
+
+static const uint16_t evdev_to_hid[276] = {
+	0x0001, 0x0029, // ESCAPE
+	0x0002, 0x001e, // 1
+	0x0003, 0x001f, // 2
```

Comment:
Instead of hardcoding the values in keys.h you should use their names (e.g KEY_A instead of 0x0001).

Reply:
Yes, looks cleaner now. Thanks.